ha/alert/alert_monitor.py

```diff
@@ -25,6 +25,8 @@
 from ha.core.config.config_manager import ConfigManager
 from ha.alert.alert_factory import AlertFactory
 from ha.alert.const import ALERTS
+from filter import AlertEventFilter
```

Comment:
Please confirm that not having full path for import of this module is not going to cause any issues ; if not already done

Reply:
yeahh we should use full path.